src/Components/CreateImageWizard/steps/review.js

```diff
@@ -0,0 +1,14 @@
+import CustomButtons from '../formComponents/CustomSubmitButtons';
+
+export default {
+    name: 'review',
+    title: 'Review',
+    // eslint-disable-next-line react/display-name
```

Comment:
I don't believe this is necessary. Or, at least, I don't see any errors when I remove it.

Reply:
Fixed by @Gundersanne in https://github.com/Gundersanne/image-builder-frontend/commit/0a214b656b4aff49856262099e049d923f001bef